mlflow/ml-package-versions.yml

```
@@ -255,6 +255,18 @@ spacy:
     run: |
       pytest tests/spacy/test_spacy_model_export.py --large
 
+spark-3.x:
+  package_info:
+    pip_release: "pyspark"
+
+  autologging:
+    minimum: "3.0.0"
+    maximum: "3.1.1"
+    run: |
+      # TODO: Add datasource autologging tests
+      pytest tests/spark_autologging/test_pyspark_ml_autologging.py --large
+      pytest tests/spark_autologging/test_pyspark_ml_autologging_custom_allowlist.py --large
```

```suggestion
      pytest --verbose tests/spark_autologging/test_pyspark_ml_autologging.py --large
      pytest --verbose tests/spark_autologging/test_pyspark_ml_autologging_custom_allowlist.py --large
```

Can we add `--verbose` to make it easier to identify which tests fail / succeeds?